src/components/BadgerBadge/BadgerBadge.js

```diff
@@ -144,6 +145,7 @@ class BadgerBadge extends React.PureComponent<Props> {
 			showAmount,
 			showQR,
 			paymentRequestUrl,
+			invoiceTimeLeftSeconds,
```

Comment:
Missing in `Props`, please add it in there to fix the `flowjs` error

Reply:
38e9062